include/sympack/symPACKMatrix2D.hpp

```diff
@@ -22,15 +32,13 @@
 #define _PRIORITY_QUEUE_RDY_
 
 
-//#define _USE_PROM_AVAIL_
-//#define _USE_PROM_RDY_
+#define _USE_PROM_AVAIL_
+#define _USE_PROM_RDY_
```

Comment:
These settings are a behavioral change for non-CUDA mode relative to current master branch.

I don't know any reason to object to this change, just wanted to note it for completeness.